package.json

```diff
@@ -62,12 +63,14 @@
     "@babel/polyfill": "^7.4.4",
     "classlist-polyfill": "^1.2.0",
     "curl": "cujojs/curl#0.8.13",
-    "d3": "^3.5.4",
+    "d3": "^6.2.0",
     "domready": "^1.0.7",
     "fastclick": "^1.0.6",
     "fence": "git://github.com/guardian/fence",
+    "index": "^0.4.0",
     "intersection-observer": "^0.11.0",
     "raf": "^3.4.0",
-    "smooth-scroll": "^16.1.3"
+    "smooth-scroll": "^16.1.3",
+    "d3-shape": "latest"
```

Comment:
Can we pin to the latest version?

Me and @dskamiotis had a bug today that came from relying on latest versions in Bridget 🙃